agent/virus_total_agent.py

```diff
@@ -106,10 +109,19 @@ def _prepare_targets(self, message: msg.Message) -> list[str]:
         from the config."""
         if message.data.get("host") is not None:
             host = str(message.data.get("host"))
-            if message.data.get("mask") is None:
+            mask = message.data.get("mask")
+            if mask is None:
                 ip_network = ipaddress.ip_network(host)
             else:
-                mask = message.data.get("mask")
+                version = message.data.get("version")
+                if version == 4 and int(mask) < IPV4_CIDR_LIMIT:
+                    raise ValueError(
```

Comment:
Why raise a value error? why not a custom error, also do we want the agent to crash if the limit is not supported?

Reply:
We use `ValueError` because it clearly indicates an unsupported value from the user. Crashing is intentional for optimal resource usage in this scenario.

Reply:
+1 to hamza comment, i dont think we need custom error here !

+ its the same logic used in other agents